src/OpenFeature.Contrib.Providers.Flipt/README.md

```diff
@@ -0,0 +1,195 @@
+# Flipt .NET Provider
+
+An OpenFeature Provider which enables the use of the Flipt Server-Side SDK.
+
+# .NET SDK usage
+
+## Install dependencies
+
+We will first install the **flipt provider**.
+
+#### .NET CLI
+```shell
+dotnet add package OpenFeature.Contrib.Providers.Flipt
+```
+#### Package Manager
+
+```shell
+NuGet\Install-Package OpenFeature.Contrib.Providers.Flipt
+```
+#### Package Reference
+
+```xml
+<PackageReference Include="OpenFeature.Contrib.Providers.Flipt" />
+```
+#### Paket CLI
+
+```shell
+paket add OpenFeature.Contrib.Providers.Flagd
+```
+
+
+## Using the Flipt Provider with the OpenFeature SDK
+
+Using the Flip provider assumes that you have installed and running Flipt service. Instructions on how to do this can be found in the [official documentation](https://docs.flipt.io/self-hosted/overview)
+
+The following example shows how to use the Flipt provider with the OpenFeature SDK.
+
+*appsettings.json*
+```json
+{
+    "OpenFeature": {
+        "Provider": {
+            "Flipt": {
+                "ServiceUri": "http://localhost:9000",
+                "Namespace": "default",
+                "TargetingKey": "UserId",
+                "RequestIdKey": "RequestId",
+                "UseBooleanEvaluation": false
+            }
+        }
+    }
+}
+```
+
+*Program.cs*
+
+```csharp
+using OpenFeature.Contrib.Providers.Flipt;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Get provider config from configuration
+var config = builder.Configuration.GetSection("OpenFeature:Provider:Flipt").Get<FliptProviderConfiguration>();
+
+/* Or create config directly:
+var config = new FliptProviderConfiguration
+{
+    ServiceUri = new Uri("http://localhost:9000"),
+    Namespace = "default",
+    TargetingKey = "UserId",
+    RequestIdKey = "RequestId",
+    UseBooleanEvaluation = false
+};*/
+
+// Create an instance of Flipt provider
+var fliptProvider = new FliptProvider(config);
+
+// Set the Flipt Provider as the provider for the OpenFeature SDK
+await OpenFeature.Api.Instance.SetProviderAsync(fliptProvider);
+
+// Create OpenFeature client with current provider
+var client = OpenFeature.Api.Instance.GetClient("my-app");
+
+// Resolve boolean flag value
+var isTestFlagEnabled = await client.GetBooleanValue("test-flag", false, null);
+
+System.Console.WriteLine($"Test flag enabled: {isTestFlagEnabled}");
+```
+
+## Boolean flag evaluation
+To perform Boolean flag evaluation, such as `GetBooleanValue` or `GetBooleanDetail` you can utilize [Boolean Evaluation](https://docs.flipt.io/reference/evaluation/boolean-evaluation) or [Variant Evaluation](https://docs.flipt.io/reference/evaluation/variant-evaluation) APIs.  
```

Comment:
this seems a bit odd, wouldnt we want to rely on boolean flags always for `GetBooleanX` calls ?